### DISCUSSION TITLE
下载 zip 包更新时如何保留历史数据，谢谢

### DISCUSSION THREAD

**Discussion by declandragon** [ronggang/PT-Plugin-Plus#1931] [Q&A]
下载 zip 包更新时如何保留历史数据，谢谢

**Comment by ted423** [comment]
我向来是推荐crx

**Reply by ted423** [reply]
pre-release的crx

**Reply by declandragon** [reply]
最开始也是使用的 crx ，添加之后无法启用，请问应该怎么处理？
![image](https://github.com/pt-plugins/PT-Plugin-Plus/assets/37643710/9aa11102-93db-4c08-a463-2bf88737adce)

**Comment by declandragon** [comment]
用这个方法可以启用了，但是更新的时候，还是需要卸载重装之类的，历史数据还是没有了

https://blog.csdn.net/NXY666/article/details/121490881

**Reply by fqh1995** [reply]
历史数据可以本地备份再回复啊

**Reply by declandragon** [reply]
> 历史数据可以本地备份再回复啊

已经使用了 pre-release的crx 安装了， 会自动更新，谢谢
